### ISSUE TITLE
20 Deep Maps

### ISSUE THREAD

**Issue by LostKing73** [cs6413110/Pixel-Tanks#227]
dk where else to put so ill put here, no need for comments, pretty much just documentation

**Comment by LostKing73** [comment]
blank code

[null,'deep','R968X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R33X27R1045']

**Comment by LostKing73** [comment]
lvl 2 - escape
![image](https://github.com/user-attachments/assets/34a401ed-89e3-4b22-b7e4-e2e74d92dbe8)
[null,'deep','R968X27R33X27R33X27R33X6QX20R33X5Q3X14ZX4R33X4Q3X7GX6Z3X3R33X5QX16ZX4R33X27R33X27R33X27R33X11I2XI2X6GX4R33X4ZX6IX3IX11R33X3Z3X5IXSXIX11R33X4ZX6IX3IX11R33X11I5X11R33X27R33X6GX16QX3R33X22Q3X2R33X15ZX5Q3X3R33X14Z3X5QX4R33X15ZX11R33X8QX18R33X7Q3X17R33X8Q3X2G3X8GX2R33X3GX5QX2G5X3GX6R33X12G2FG2X10R33X12G5X10R1045']

**Comment by cs6413110** [comment]
Like you were expecting some hard, intense pve levels and then you just get super fast Minecraft ancient city gameplay. Also literally like no rewards since no AI :(

**Comment by LostKing73** [comment]
> Ok, so ik these are supposed to be leviathan levels since there is no AI, but like.... won't people get bored since late game is just anti-pve? Like, imagine, you love playing singleplayer and fighting AI and practicing your combos on them and then the last 20 levels are just some horror game where your goal is to speedrun mining blocks to get out.

there not the last 20, gem is after them, and you can still do the other ones, and you get xp if u kill leviathan, its just not rly recommended cuz it makes it harder

**Comment by LostKing73** [comment]
> LostKing leviathan is ded. Check the wiki page for info on what level ai will be, and how many u should be adding

no, there's still a chance for it. >:( I'm not giving up on it

**Comment by BreadSpammer** [comment]
k I'll just make deep maps too ig

**Comment by LostKing73** [comment]
> k I'll just make deep maps too ig

no, actually. when all of the other ones are done, and if cs cant start coding leviathan, then we'll split it 50/50. i just dont rly want it to be the same kind of gameplay loop the previous like 60 level would have had, so i wanna wait for leviathan

**Comment by BreadSpammer** [comment]
but like leviathan is ded

**Comment by LostKing73** [comment]
> but like leviathan is ded

no, it WILL be IF we don't have time at the end. its just EXPECTED to be dead.

**Comment by LostKing73** [comment]
theres a 1 in a billion chance I learn javascript and just code it myself

**Comment by LostKing73** [comment]
lvl 3 - Survive
![image](https://github.com/user-attachments/assets/11f699d0-4060-45bf-8df1-7c7c203fa9a7)
[null,'deep','R968X19I2X6R33X19I2X6R33X19I2X6R33X19I2X6R33Z6IZ12G2Z6G7R26Z6IZ12G2Z6G7R26X19Z2X6R5G2R26X19Z2X6R5G2R26X19Z2X6R5G2R26X19Z2X6R5G2R26X6IX12Z2X6R5G2R26X6IX12Z2X6R5G2R26X4I5X10Z2X6R5G2R26X4IX3IX10Z2X6R5G2R26X4IXSX12Z2X6R5G2R26X4IX3IX10Z2X6R5G2R26X4I5X10Z2X6R5G2R26X6IX12Z2X6R5G2R26X6IX12Z2X6R5G2R26X19Z2X6R5G2R26X19Z2X6R5G2R26X19Z2X6R5G2R26X19Z2X6R5G2R26X6IX12Z2X6R5G2R26Z6IZ12G2Z6G7R26Z6IZ12G2Z6G7R26X6IX12Z2X6R1045']
